src/scripts/clipperUI/panels/successPanel.tsx

```diff
@@ -9,12 +9,27 @@ import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
 import {Clipper} from "../frontEndGlobals";
+import {RatingsHelper, RatingsPromptStage} from "../ratingsHelper";
 import {Status} from "../status";
 
 import {SpriteAnimation} from "../components/spriteAnimation";
 
-class SuccessPanelClass extends ComponentBase<{ }, ClipperStateProp> {
-	onLaunchOneNoteButton() {
+import {DialogButton, DialogPanel} from "./dialogPanel";
+
+interface SuccessPanelState {
+	currentRatingsPromptStage?: RatingsPromptStage;
```

Comment:
This makes sense. In other words, Instead of the ratings prompt being a sub component of the successPanel, the components are separate and just shown on the same panel? An analagous component are the modeButtons